laboratory/views/organizations.py

```diff
@@ -0,0 +1,122 @@
+# encoding: utf-8
+'''
+Created on 26/12/2016
+
+@author: luisza
+'''
+from __future__ import unicode_literals
+
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import get_object_or_404
+from django.urls.base import reverse_lazy
+from django.utils.decorators import method_decorator
+from django import forms
+from mptt.forms import TreeNodeChoiceField
+from django.utils.functional import lazy
+from django.db.models import Q
+from laboratory.models import LaboratoryRoom, Laboratory, OrganizationStructure, PrincipalTechnician
+from laboratory.decorators import check_lab_permissions, user_lab_perms
+
+from .djgeneric import  ListView
+
+
+
+
+
+class OrganizationSelectableForm(forms.Form):
+    organizations = OrganizationStructure.objects.none()
+    filter_organization= TreeNodeChoiceField(queryset=organizations)
+
+    
+    def __init__(self,user, *args, **kwargs):
+        self.user = user 
+        super(OrganizationSelectableForm, self).__init__(*args, **kwargs)  
+        organizations = OrganizationStructure.objects.filter(
+            principaltechnician__credentials=self.user)
+        
+        orgs = None
+        for org in organizations:
+            if orgs is None:
+                orgs = Q (pk__in=org.get_descendants(include_self=True))
+            else:
+                orgs |= Q (pk__in=org.get_descendants(include_self=True) )   
+        
+        
+        if organizations.exists():               
```

Comment:
orgs is not None is better, because organizations.exists() require more internal checks than  a logic operator "None"